plugin-dev/Source/Sentry/Public/SentrySettings.h

```diff
@@ -108,6 +104,22 @@ class SENTRY_API USentrySettings : public UObject
 		Meta = (DisplayName = "Attach game log to captured events", ToolTip = "Flag indicating whether to attach game log automatically to captured events. Not available in shipping builds."))
 	bool EnableAutoLogAttachment;
 
+	UPROPERTY(Config, EditAnywhere, Category = "Release & Health",
+		Meta = (DisplayName = "Enable automatic session tracking ", ToolTip = "Flag indicating whether the SDK should automatically start a new session when it is initialized."))
+	bool EnableAutoSessionTracking;
+
+	UPROPERTY(Config, EditAnywhere, Category = "Release & Health",
+		Meta = (DisplayName = "Session timeout, milliseconds (for Android/Apple only)", ToolTip = "Time interval after which session will be terminated if application is in background mode."))
+	int32 SessionTimeout;
+
+	UPROPERTY(Config, EditAnywhere, Category = "Release & Health",
+		Meta = (InlineEditConditionToggle))
+	bool OverrideReleaseName;
```

Comment:
I'm confused by this. If it's not overridden the SDK will get the ReleaseName from the SDK instead of the settings?